Automattic/woocommerce-payments
# Fee discounts should use the discount currency rather than the base fee currency

Fixes #2371

#### Changes proposed in this Pull Request

Fee discounts displayed on the overview page should use the discount currency rather than the base fee currency.

e.g. You have a store in the UK with GBP set as the store currency. You have a fee discount applied to your account for $25,000 USD. This discount should be displayed as USD on the overview page rather than GBP.

| Before | After |
|-|-|
|<img width="695" alt="Screen Shot 2021-07-20 at 2 18 54 pm" src="https://user-images.githubusercontent.com/1527164/126261777-c50df839-118b-443d-8da1-67875a273894.png">|<img width="699" alt="Screen Shot 2021-07-20 at 2 15 12 pm" src="https://user-images.githubusercontent.com/1527164/126261787-70c809d4-494c-407f-84f8-4a8dac155bd2.png">|

#### Testing instructions

* Reonboard your store a non-USD merchant.
* Assign a promo to your test account via manual DB query (I used the `wcpay-promo-2021-6-incentive-2` promo)
* Refresh the account cache on the client-side in WCPay Dev and ensure **Payments → Overview** shows the progress bar and discount description representing the volume in USD.

-------------------
